assets/js/builder/Schemas/Quiz.js

```diff
@@ -67,8 +67,21 @@ define( [], function() {
 						label: LLMS.l10n.translate( 'Randomize Question Order' ),
 						tip: LLMS.l10n.translate( 'Display questions in a random order for each attempt. Content questions are locked into their defined positions.' ),
 						type: 'switch',
+						condition: function() {
+							return 'yes' === this.get( 'question_bank' ) ? false : true;
+						}
 			},
-				],
+				], [
+					{
+						attribute: 'number_of_questions',
```

Comment:
why the attribute here is `number_of_questions`?

Reply:
Left by mistake.

It's of no use in this case. `id` can also be removed.